test/token/ERC1404/ERC1404.behavior.js

```diff
@@ -19,6 +21,8 @@ const describeBehaviorOfERC1404 = function ({ deploy, restrictions, name, symbol
     // eslint-disable-next-line mocha/no-setup-in-describe
     describeBehaviorOfERC1404Base({
       deploy,
+      mint,
```

Comment:
Passing `mint` and `burn` is not necessary here, because the `skips` array includes `::ERC20Base`.